IndividualSpreadFooting.tex

```diff
@@ -18,10 +18,10 @@ \section{Introduction} Clause 34 of the  \citetitle{IS2000} gives the
 
 \item 25\% excess pressure on edge of footing was allowed by the 
 \citetitle{IS1964} when a footing is eccentrically loaded. This 
-concession is withdrawn by the \citetitle{IS2000}, thereby adding to 
+concession is withdrawn by the ,\citetitle{IS2000} thereby adding to 
```

Comment:
No, we don't need this comma, By mistake it written. I will  remove it in next PR.